app/javascript/components/close_contact/CloseContact.js

```diff
@@ -132,17 +140,46 @@ class CloseContact extends React.Component {
             </Row>
             <Row>
               <Form.Group as={Col} controlId="email">
-                <Form.Label className="nav-input-label">Email</Form.Label>
+                <Form.Label className="nav-input-label">Email {schema?.fields?.email?._exclusive?.required && ' *'} </Form.Label>
                 <Form.Control size="lg" className="form-square" value={this.state.email || ''} onChange={this.handleChange} />
                 <Form.Control.Feedback className="d-block" type="invalid">
                   {this.state.errors['email']}
                 </Form.Control.Feedback>
               </Form.Group>
             </Row>
+            <Row>
+              <Form.Group as={Col} controlId="last_date_of_exposure">
+                <Form.Label className="nav-input-label">Last Date of Exposure {schema?.fields?.last_date_of_exposure?._exclusive?.required && ' *'}</Form.Label>
+                <DateInput
+                  id="last_date_of_exposure"
+                  date={this.state.last_date_of_exposure}
+                  minDate={'1900-01-01'}
+                  maxDate={moment().format('YYYY-MM-DD')}
+                  onChange={this.handleDateChange}
+                  placement="top"
+                  isInvalid={!!this.state.errors['last_date_of_exposure']}
+                  customClass="form-control-lg"
+                  ariaLabel="Last Date of Exposure Input"
+                />
+                <Form.Control.Feedback className="d-block" type="invalid">
+                  {this.state.errors['last_date_of_exposure']}
+                </Form.Control.Feedback>
+              </Form.Group>
+            </Row>
+            <Row>
+              <Form.Group as={Col} controlId="assigned_user">
```

Comment:
Can you add the datalist of existing assigned users that shows up for all the other assigned user inputs? I think you should be able to just copy/paste most of it

Reply:
also, this input should have pretty strict validation (only numeric and only 1-9999), just take a look at how it's done in other places

Reply:
I like this! I've updated the validation, and added the datalist. Wasnt quite a copy and paste, but essentially the same thing.